torchbenchmark/models/pytorch_CycleGAN_and_pix2pix/__init__.py

```diff
@@ -35,16 +35,18 @@ def __init__(self, test, device, batch_size=None, extra_args=[]):
         results_arg = f"--results_dir {results_dir}"
         data_root = os.path.join(DATA_PATH, "pytorch_CycleGAN_and_pix2pix_inputs")
         device_arg = ""
+        device_type_arg = f"--device_type {self.device}"
         if self.device == "cpu":
             device_arg = "--gpu_ids -1"
-        elif self.device == "cuda":
+        else:
```

Comment:
@weishi-deng , do you know if it still needs to support other devices like mps?

Reply:
We don't have mps platforms for test but this PR will also cover those devices as long as they support these APIs: "torch. device.is_available()", "torch.device("device:index")", "tensor.to("device:index".